tests/utils.py

```diff
@@ -138,6 +138,8 @@ def markdown_checker_configs(tmpdir):
             {"pattern": "http://127.0.0.1:5000"},
             {"pattern": "https://adb-3214.67.azuredatabricks.net*"},
             {"pattern": "https://adb-345.89.azuredatabricks.net*"},
+            {"pattern": "https://adb-xxxx.xx.azuredatabricks.net*"},
```

Comment:
Why are we adding these in?

Reply:
It was originally for the test workspace, now removed. So I will remove this too.